frontend/src/components/AlterSchemaPrepForm/ProjectTenantView.vue

```diff
@@ -88,22 +88,22 @@
 /* eslint-disable vue/no-mutating-props */
 
 import { computed, watchEffect, watch, ref } from "vue";
-
-import {
+import { NCollapse, NCollapseItem } from "naive-ui";
+import { groupBy } from "lodash-es";
+import type {
   Database,
   DatabaseId,
   Environment,
   LabelKeyType,
   Project,
-  UNKNOWN_ID,
-} from "../../types";
-import { NCollapse, NCollapseItem } from "naive-ui";
-import { groupBy } from "lodash-es";
+} from "@/types";
+import { UNKNOWN_ID } from "@/types";
```

Comment:
```suggestion
  UNKNOWN_ID,
} from "@/types";
```

Reply:
I've got no idea why sometimes when we import a type from `@/types` directly (without `import type {xxx} from "@/types"`) the vue compiler toolchain complains. So I split type imports and variable/constant imports.